master/buildbot/process/build.py

```diff
@@ -89,6 +90,7 @@ def __init__(self, requests):
         self.terminate = False
 
         self._acquiringLock = None
+        self.result = SUCCESS   # overall result, may downgrade after each step
```

Comment:
You're right that we usually use "results" and should keep that up.  It's not quite right in English, but better to be consistent with the rest of the codebase.